services/nomad/build/build-rsyncd.nomad

```diff
@@ -0,0 +1,57 @@
+job "build-rsyncd" {
+  type = "service"
+  datacenters = ["VOID"]
+  namespace = "build"
+
+  group "rsync" {
```

Comment:
This group is missing a [service definition](https://developer.hashicorp.com/nomad/docs/job-specification/service).